src/java/org/apache/cassandra/db/ReadCommand.java

```diff
@@ -578,12 +578,8 @@ private void countTombstone(ClusteringPrefix<?> clustering)
             public void onClose()
             {
                 recordReadLatency(metric, System.nanoTime() - startTimeNanos);
-
-                metric.tombstoneScannedHistogram.update(tombstones.get());
-                metric.liveScannedHistogram.update(liveRows);
-
-                if (tombstones.checkAndTriggerWarning())
-                    metric.tombstoneWarnings.inc();
+                metric.incLiveRows(liveRows);
+                metric.incTombstones(tombstones.get(), tombstones.checkAndTriggerWarning());
```

Comment:
This is the important part for the CNDB `SingleCompactorTest.garbageCollectionTest`